src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1031,6 +1031,18 @@ void CodeGen::genSSE2Intrinsic(GenTreeHWIntrinsic* node)
             break;
         }
 
+        case NI_SSE2_StoreNonTemporal:
```

Comment:
Why can't this be handled by the table driven 2-op MemoryStore case?

Reply:
Essentially it is a scalar intrinsic and @fiigii asked specifically to put it in this category. Currently scalar intrinsics are not table driven.

Reply:
https://github.com/dotnet/coreclr/pull/16832#discussion_r173096844

Reply:
Ah, I see. We can fix this later.
Thanks

Reply:
> . We can fix this later.

Implementing `Bmi1/2` is a good opportunity.